corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -87,35 +88,36 @@ def add_arguments(self, parser):
         parser.add_argument('--delimiter', default='\t', choices=('\t', ','))
 
     def handle(self, domain, data_models, delimiter, **options):
-        data_models = set(data_models)
+        with mock.patch('sys.stdout', self.stdout):
```

Comment:
Is this needed for tests only, or is it also important for operation of this command outside of tests? If tests only then this should be moved into the test code. Probably this
https://github.com/dimagi/commcare-hq/blob/0dec6fb5fc76968ccb91f77516aa4f73bf7ed639/corehq/apps/hqadmin/tests/test_stale_data_in_es.py#L99
should be replaced with
```py
with mock.patch('sys.stdout', f):
     call_command('stale_data_in_es', *args, stdout=f, **kwargs)
```

Reply:
Oh, yeah that's brilliant. I was thinking you're supposed to always use `self.stdout` instead of `sys.stdout` in management commands, but the only practical reason for that that I'm aware of at least in this case is for tests. So I'll go ahead and take your suggestion